tools/lint/lint.py

```diff
@@ -957,6 +957,8 @@ def create_parser():
                         "working directory, not just files that changed")
     parser.add_argument("--github-checks-text-file", type=ensure_text,
                         help="Path to GitHub checks output file for Taskcluster runs")
+    parser.add_argument("--jobs", "-j", type=int, default=0,
+                        help="Level to parallelism to use (0 detects CPUs)")
```

Comment:
Might as well
```suggestion
                        help="Level to parallelism to use (defaults to the number of CPUs)")
```
since you can't see 0 is the default in the help message